lib/install/config/webpacker.yml

```diff
@@ -35,7 +36,13 @@ test:
 
   public_output_path: packs-test
 
+  compile_missing_packs: true
+
+  compiler_cache_path: tmp/.webpacker-checksum
```

Comment:
Doesn't need to be configurable, IMO. How about `tmp/webpacker/checksum`? Then any future temp files can share that dir.